nodeset/Opc.Ua.PlasticsRubber.GeneralTypes.NodeSet2.xml

```diff
@@ -0,0 +1,14109 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ * Copyright (c) 2005-2022 The OPC Foundation, Inc. All rights reserved.
+ *
+ * OPC Foundation MIT License 1.00
+ * 
+ * Permission is hereby granted, free of charge, to any person
+ * obtaining a copy of this software and associated documentation
+ * files (the "Software"), to deal in the Software without
+ * restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following
+ * conditions:
+ * 
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+ * OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+ * WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+ * OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * The complete license agreement can be found here:
+ * http://opcfoundation.org/License/MIT/1.00/
+-->
+
+<UANodeSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:uax="http://opcfoundation.org/UA/2008/02/Types.xsd" xmlns="http://opcfoundation.org/UA/2011/03/UANodeSet.xsd" xmlns:s1="http://opcfoundation.org/UA/PlasticsRubber/GeneralTypes/Types.xsd" xmlns:s2="http://opcfoundation.org/UA/DI/Types.xsd" xmlns:ua="http://unifiedautomation.com/Configuration/NodeSet.xsd" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+    <NamespaceUris>
+        <Uri>http://opcfoundation.org/UA/PlasticsRubber/GeneralTypes/</Uri>
+        <Uri>http://opcfoundation.org/UA/DI/</Uri>
+    </NamespaceUris>
+    <Models>
+        <Model ModelUri="http://opcfoundation.org/UA/PlasticsRubber/GeneralTypes/" PublicationDate="2022-07-13T00:00:00Z" Version="1.03.01-RC">

```

Comment:
@GoetzGoerisch  Could you give it a try and replace `Version="1.03.01-RC">` with `Version="1.03.01`?
It could be that the string may cause trouble.
Wasn't an exception being caught?

Reply:
Thank you @swamper123 but only the required version changes in the output. No other exception is logged.

https://github.com/umati/Sample-Server-asyncio/pull/40/commits/395c281faec753bd651e546f244b2641f518e619

`WARNING:asyncua.common.xmlimporter:Model is missing: http://opcfoundation.org/UA/PlasticsRubber/GeneralTypes/ - Version: 1.03.01 - PublicationDate: 2022-07-13T00:00:00Z or newer`

Reply:
Since caught exceptions are just printed, nothing appeared as well?

What do you get if you change the `print(e)` to `logging.warning("Import Error - {}".format(e))` or `logging.error("Import Error - {}".format(e)))`?

https://github.com/umati/Sample-Server-asyncio/blob/395c281faec753bd651e546f244b2641f518e619/src/server.py#L123

Reply:
see below

Reply:
Okay, it is still the same issue from ~ last year: https://github.com/FreeOpcUa/opcua-asyncio/issues/655
I bumped the problem again.